.devcontainer/devcontainer.json

```diff
@@ -58,6 +59,7 @@
 		// Run the entrypoint defined in container image.
 		"--init"
 	],
+    "postCreateCommand": "npm install -g @devcontainers/cli",
```

Comment:
Totally a nit, but this is indented using spaces and the rest uses tabs

Reply:
Good spot - fixed. While I was there I also tweaked the config to remove the schema validation warnings in `devcontainer.json` 😄